src/shared/seccomp-util.c

```diff
@@ -1192,6 +1192,27 @@ int seccomp_restrict_realtime(void) {
         return 0;
 }
 
+static int add_seccomp_syscall_filter(scmp_filter_ctx seccomp,
```

Comment:
Eeeh, I don't think this is better. After all it's a static function that already has a fairly long name.